collect_app/src/test/java/org/odk/collect/android/utilities/PermissionsTest.kt

```diff
@@ -0,0 +1,59 @@
+package org.odk.collect.android.utilities
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Description
+import org.hamcrest.StringDescription
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.manifest.AndroidManifest
+import org.robolectric.res.Fs
+
+/**
+ * Test for checking permissions in [AndroidManifest]
+ */
+@RunWith(AndroidJUnit4::class)
+class PermissionsTest {
```

Comment:
I'm not sure if I see the need for this test. It feels like we're just testing config right?

Reply:
I also don't think we need this so ok let's remove it.